### ISSUE TITLE
Lint is not working for catching Namespace Indentation error

### ISSUE THREAD

**Issue by Yujinmon** [cpplint/cpplint#234]
Hello, i found some error related to indentation and it seems worth it to solve.

In google style guide, there is the contents for namespace indentation
https://google.github.io/styleguide/cppguide.html#Namespace_Formatting
Specifying 'the contents of namespace are not indented'.

For example,
<img width="441" alt="image" src="https://github.com/cpplint/cpplint/assets/78896558/58aa7e2f-3a49-4206-ad6f-a3c4df9f3b8f">
You can see there is indent inside in namespace scope.
This is not fit to google style guide.

<img width="433" alt="image" src="https://github.com/cpplint/cpplint/assets/78896558/64c9716b-4f56-4bc7-bf76-143aa4fe0084">
This is the example of correct style refer to google style guide.

And as you can see, 
<img width="843" alt="image" src="https://github.com/cpplint/cpplint/assets/78896558/ee17f3dd-4911-4a4c-ad0d-0b750fb4503e">
lint is not catching!

I think it would be great that if this problem solved.
Can anyone give comments or feedback for this issue?

Thankyou!

**Comment by Yujinmon** [comment]
There is a patch for this error, and i'm preparing for pull request.
If anyone interested, please check the pr!

**Comment by jiawen** [comment]
Since the update to 2.0, I'm seeing the opposite. It's overly aggressive in complaining that I content within a namespace should not be indented even though it's not (a false positive). E.g.,

```
#include "gtest/gtest.h"

namespace MyNamespace {
namespace {

TEST(MyTest,
     SomeVeryLongTestNameSoThatClangFormatBreaksLinesAtEightyChars) {
  // Test body.
}

}  // namespace
}  // namespace MyNamespace
```

cpplint complains that the line starting with `SomeVeryLong...`, should not be indented.

**Comment by aaronliu0130** [comment]
@jiawen This is an old and fixed issue, and you're correct that we now have #293.

**Comment by jiawen** [comment]
> [@jiawen](https://github.com/jiawen) This is an old and fixed issue, and you're correct that we now have [#293](https://github.com/cpplint/cpplint/issues/293).

Ha I must be blind. I thought I looked through all issues with `namespace` in the title. Anyway, thanks! Looks like you just fixed #293.
